### PR TITLE
JAMES-2349 Add user quota endpoints in WebAdmin documentation

### PR DESCRIPTION
**Pull request by aduprat**

### CODE REVIEW

**Comment by chibenwa** [comment]
No doc for `quota details`?

File: src/site/markdown/server/manage-webadmin.md [hunk 1]
```diff
@@ -238,7 +238,23 @@ curl -XGET http://ip:port/quota/users/usernameToBeUsed
 
 Resource name usernameToBeUsed should be an existing user
 
-The answer can contain a fixed value, an empty value (null) or an unlimited value (-1):
+The answer is the details of the quota of that user.
+
+```
+{
+  "global": {"count":252,"size":242}
```
Comment (aduprat) [inline src/site/markdown/server/manage-webadmin.md]: comma is missing at the end of the line (also on others)

Comment (chibenwa) [reply src/site/markdown/server/manage-webadmin.md]: Well  spotted. I'll correct it.

File: src/site/markdown/server/manage-webadmin.md [hunk 2]
```diff
@@ -227,124 +227,192 @@ Response codes:
...
-Response codes:
- - 200: Nothing special
- - 500: Internal error
+ - The `global` entry represent the quota limit allowed on this James server.
+ - The `domain` entry represent the quota limit allowed for the user of that domain.
+ - The `user` entry represent the quota limit allowed for this specific user.
+ - The `computed` entry represent the quota limit applied for this user, resolved from the upper values.
 
-### Updating per quotaroot mail count limitation
+Note that `quota` objectcan contain a fixed value, an empty value (null) or an unlimited value (-1):
```
Comment (rouazana) [inline src/site/markdown/server/manage-webadmin.md]: missing space between object and can

**Comment by chibenwa** [comment]
merged

**Pull request closed without merging**